packages/react-admin-core/src/FinalForm.sc.ts

```diff
@@ -1,7 +1,3 @@
 import styled from "styled-components";
```

Comment:
Das File braucht man so (ohne style-definitions) nicht mehr.

Reply:
@rabengraph ist schon in #175 gemacht